pkg/cmd/cmdutil/cmdutil.go

```diff
@@ -142,3 +141,63 @@ func getHandlerPairs(cmd *cobra.Command, config cfg.Config) []handlerPair {
 
 	return handlerPairs
 }
+
+// registerStatic registers handlers for ALL static paths
+func registerStatic(config cfg.Config) {
```

Comment:
This all stems from using the `mux.NewRouter()` for the path registrations/bindings, which is defined at the top of `cmdutil.go`. Initially, I wanted this all to be defined in `httpserver.go` , but ran into cyclic dependency issue. 

Since `cmdutil.go` had _ RegisterHandlers()_ already defined I thought I'd move it all here. I agree it's awkward; open to discuss alternatives offline